getBagsFunction.php

```diff
@@ -0,0 +1,13 @@
+<?php
+    /**
+         * fetches all bags
+         * 
+         * @return
+         */
+
+        function getBags(&$database) {
```

Comment:
I'd remove the &, that means the PDO object is passed as reference rather than a copy, but it isn't actually doing anything in this particular case.

Reply:
Also add parameter and return type hinting